ktorm-support-mysql/src/main/kotlin/me/liuwj/ktorm/support/mysql/MySqlDialect.kt

```diff
@@ -116,6 +121,21 @@ open class MySqlFormatter(database: Database, beautifySql: Boolean, indentSize:
         write(") ")
     }
 
+    private fun visitColumnAssignmentsWithValues(original: ArrayList<ColumnAssignmentExpression<*>>):
```

Comment:
Why add this function? Is it different from the existing function `visitColumnAssignments`?

Reply:
@vincentlauvlwj Removed.
For the first time, I couldn't understand how to transform expressions to sql. This is just remains.